test/monitoringe2e/helpers.go

```diff
@@ -159,7 +157,13 @@ func VerifyMetrics(caCtx *test.Context, metricQueries []string) error {
 				caCtx.T.Log("Error querying prometheus metrics:", err)
 				return false, nil
 			}
-			return value.Type() == prommodel.ValVector, nil
```

Comment:
I guess we can expand this for scalar too as in [here](https://github.com/prometheus/client_golang/issues/194#issuecomment-254443951) but I guess we are good for now.